src/codegen/dispatch.rs

```diff
@@ -276,6 +281,7 @@ fn add_dispatch_case(
 /// Create the dispatch for a contract constructor. This case needs creates a new function in
 /// the CFG because we want to use de abi decoding implementation from codegen.
```

Comment:
```suggestion
/// Create the dispatch for a contract constructor. This case creates a new function in
/// the CFG because we want to use the abi decoding implementation from codegen.
```